day13/cruise-control.py

```diff
@@ -0,0 +1,20 @@
+def maxSpeed(horses, totalDist):
+    # for n horses, find the times of all horses,
+    # then get the slowest time and divide totalDist by that time
+    largestTime = 0.0
+
+    for horse in horses:
+        start, speed = horse
+        time = (totalDist-start)*1.0/speed
```

Comment:
add a comment or just wrap `(totalDist-start)` in a call to `float()`. otherwise people will not know why it's there